src/components/SEO.js

```diff
@@ -8,24 +8,33 @@ export default function SEO ({
   twitterHandle,
   favicon,
   ogImage,
-  url
+  url,
+  fonts = [
+    {
+      url:
+        'https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap'
+    }
+  ]
 }) {
+  const Fonts = () =>
+    fonts.map(font => <link href={font.url} rel='preload' as='font' />)
+
   return (
     <Head>
       <title>{title}</title>
+
+      <link rel='icon' href={favicon} />
```

Comment:
Is there any technical reason to move the favicon up here, or is this for developer readability?

Reply:
I just moved it for readability to separate the links from the meta tags